### DISCUSSION TITLE
Sharing Music and Cue Point Settings Between Stations

### DISCUSSION THREAD

**Discussion by wppronto** [AzuraCast/AzuraCast#6980] [Q&A]
I'd like to run multiple stations on one server / AzuraCast instance. There will be some overlap (songs that will appear on multiple stations). Is there any way to share those among stations? The goal is not to have to enter cue points multiple times for the same songs. If we can't share the music, does the bulk import / export somehow provide a way to copy the cue points from the same songs to their counterparts on another station?

**Comment by unudelaradio** [comment]
Alternative 1: Just export the .csv file from station 1, and import it to station 2. Make sure that files on station 2 (the target files) have the same name and same path as files from station 1. When importing the csv, all fields will be updated on that target files on station 2, including cue points and also !playlists assigns! If you want to avoid changes for specific fields, just delete the csv columns you don't want. 

Alternative 2: To avoid music duplicates and save disk space, yes, you can share Media Folder from station 1 as Media Storage Location for station 2. To do that, go to Edit Station Profile for station 2 and in the Administration tab change the Media Storage Location. All media files and folders from station 1 will be visible on station 2. Be careful, any files or folders changes will always affect and always be visible on both stations. You can't share only some files or just a part from station 1 Media Folder. The entire Media Folder will become the storage folder for both stations.

**Reply by wppronto** [reply]
Thank you! And if I use option 2, are the cue points shared as well or just the raw files?

**Reply by wppronto** [reply]
Actually I guess I should try that before asking! Will do.

**Reply by unudelaradio** [reply]
Cue Points are shared as well. I think this is the best way of working. If you are the only user on the instance, sharing is the best option.

When you have multiple users on the instance, i would say that sharing between stations is not indicated.

**Comment by wppronto** [comment]
Thank you so much! Yes, I'll be the only user and only want to share among stations on once instance. Thank you for all of your help!

**Comment by Moonbase59** [comment]
I use a central song repository (on a TrueNAS server) which has the main main music folder(s) mounted via NFS into the AzuraCast server. So this can be used on multiple stations `media:` folders.

To achive that, I simply mount the main music folders on the AzuraCast host, then cloned some stations, stating I wish to keep the media. These stations can now access the main music (and the file settings), but still have their own set of files (like packaging, jingles, sweepers, etc.) in their own folders.
